docs/support/faq/how-to-use-pouta-for-remote-desktop.md

```diff
@@ -115,21 +121,22 @@ You can start the noVNC client by running
 /var/lib/snapd/snap/bin/novnc --listen 6081 --vnc localhost:5901
 ```
 
-The location of the application might be in a different location based on what
+The location of the application might be in a different based on what
```

Comment:
Maybe this is better "The application location might differ based on the Linux distribution you are using. "